recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
     - fsspec
     - python >=3.7
     - ujson
+    - zarr
```

Comment:
The metadata also has numcodecs as a dependency. I guess `zarr` brings that in but it is nice to specify just in case.